app/views.py

```diff
@@ -7,7 +7,7 @@
 
 
 # Create your views here.
-from app.models import Patient
+from app.models import User
 
 
 def register_page(request):
```

Comment:
We should add the role field here and in the register.html?

Reply:
Eventually yes. Maybe have a link that asks if the user is a employee of the lab and redirects him to a new register page? Or just add it as a drop down in this register page.